src/genie/libs/parser/iosxr/show_bgp.py

```diff
@@ -2513,7 +2513,9 @@ def cli(self, vrf_type='all', vrf='all', instance='all', neighbor='', address_fa
                             '(?P<instance>[a-zA-Z0-9\-\_\']+)$')
         p2 =  re.compile(r'^\s*BGP +neighbor +is +(?P<neighbor>[a-zA-Z0-9\.\:]+)$')
         p2_1 =  re.compile(r'^\s*BGP +neighbor +is +(?P<neighbor>[a-zA-Z0-9\.\:]+), +vrf +(?P<vrf>\S+)$')
-        p3 = re.compile(r'^Remote +AS +(?P<remote_as>[0-9]+), +local +AS +(?P<local_as_as_no>[0-9]+)(?:, +(?P<local_as_no_prepend>no-prepend))?(?:, +(?P<local_as_replace_as>replace-as))?(?:, +(?P<local_as_dual_as>dual-as))?(?:, +(?P<link_state>[a-zA-Z\s]+))?$')
+
+        # Remote AS 65107.65107, local AS 65108.65108, external link
+        p3 = re.compile(r'^Remote +AS +(?P<remote_as>[0-9\.]+), +local +AS +(?P<local_as_as_no>[0-9\.]+)(?:, +(?P<local_as_no_prepend>no-prepend))?(?:, +(?P<local_as_replace_as>replace-as))?(?:, +(?P<local_as_dual_as>dual-as))?(?:, +(?P<link_state>[a-zA-Z\s]+))?$')
```

Comment:
Can you please break the pattern in multiple lines ? It is more readable and easier to modify if necessary